iroh-ctl/src/store.rs

```diff
@@ -0,0 +1,167 @@
+use std::path::PathBuf;
+
+use anyhow::Result;
+use cid::Cid;
+use clap::{Args, Subcommand};
+use iroh_rpc_client::Client;
+
+#[derive(Args, Debug, Clone)]
+pub struct Store {
+    #[clap(subcommand)]
+    command: StoreCommands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum StoreCommands {
+    #[clap(about = "Version of the iroh store binary")]
+    Version,
+    Block(Block),
+    Dag(Dag),
+    #[clap(hide = true)]
+    GetLinks {
+        cid: Cid,
+    },
+}
+
+#[derive(Args, Debug, Clone)]
+#[clap(about = "Interact with raw IPFS blocks")]
+pub struct Block {
+    #[clap(subcommand)]
+    command: BlockCommands,
+}
+
+#[derive(Subcommand, Debug, Clone)]
+pub enum BlockCommands {
+    #[clap(about = "Get a raw IPFS block.")]
+    Get { cid: Cid },
+    #[clap(
+        about = "Store input as an IPFS block.
+Not yet implemented.",
+        hide = true
+    )]
+    Put { path: PathBuf },
+    #[clap(
+        about = "Remove IPFS block(s).
+Not yet implemented.",
+        hide = true
+    )]
+    Rm {
+        cids: Vec<String>,
+        #[clap(short, long)]
+        force: bool,
+    },
+    #[clap(hide = true)]
+    Has { cid: Cid },
+}
+
+#[derive(Args, Debug, Clone)]
+#[clap(
+    about = "Interact with IPLD DAG objects.
```

Comment:
we don't really support IPLD in that sense at the moment, so I think the names here are a bit off

Reply:
What do you think is the correct way to phrase this? "Interact with DAG objects"? "Interact with IPLD"?

Reply:
I am not sure, maybe we should just leave this part of the api off until we have made a decision around what we want to actually support there